VMBackup/main/handle.py

```diff
@@ -87,12 +88,14 @@ def main():
                 update()
             elif re.match("^([-/]*)(daemon)", a):
                 daemon()
+            elif re.match("^([-/]*)(seqNo)", a):
```

Comment:
will it match as there is a suffix as well ? Doesnt the regex need some change to allow * at end ?

Reply:
It matches, I tested it.  putting * means 0 or more times.

Reply:
ok